official_models/ev3/education_expansion/elephant/main.py

```diff
@@ -38,20 +40,20 @@ def reset():
     neck_motor.run(750)
     while color_sensor.color() != Color.RED:
         wait(10)
-    neck_motor.stop(Stop.BRAKE)
+    neck_motor.hold()
```

Comment:
`brake()` ?

Reply:
We discussed removing `brake` altogether, so I removed all uses to see if we can get away with not actually using it or if it affects the models behavior too much.